templates/sshd_config.j2

```diff
@@ -6,4 +6,6 @@ Match Group {{ sftp_group_name }}
     AllowTCPForwarding no
     X11Forwarding no
     ForceCommand internal-sftp
+    {% if sftp_allow_passwords %}PasswordAuthentication yes{% endif %}
```

Comment:
@AMeng Thanks for the pull request :)  I think the explicit setting of `PasswordAuthentication` to `no` via the `else` statement makes a lot of sense and ensures that the correct behaviour is applied throughout.  Would be great if you could make that change please.